airbyte-integrations/connectors/source-convex/source_convex/source.py

```diff
@@ -0,0 +1,160 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+
+from datetime import datetime
+from typing import Any, Iterable, List, Mapping, MutableMapping, Optional, Tuple
+
+import requests
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import IncrementalMixin, Stream
+from airbyte_cdk.sources.streams.http import HttpStream
+from airbyte_cdk.sources.streams.http.auth import TokenAuthenticator
+
+
+def convex_url_base(instance_name) -> str:
+    return f"https://{instance_name}.convex.cloud"
+
+
+# Source
+class SourceConvex(AbstractSource):
+    def _json_schemas(self, config) -> requests.Response:
+        instance_name = config["instance_name"]
```

Comment:
Do we want to use instance_name or something more user facing like deployment name or something?

Reply:
i think instance_name works for now. you need to have instance name if you're using our HTTP client (/udf and such), and this is similar. if we change the URLs for our HTTP calls, we can update this too to use team & project slug or something.